lib/supa/builder.rb

```diff
@@ -21,6 +21,19 @@ def initialize(subject, representer:, tree:)
       end
     end
 
+    def attributes(*args)
+      options = if args[-1].is_a?(Hash)
```

Comment:
@Strech  https://github.com/distribusion/supa/pull/21/commits/b3bec0a71731cce71209df8634b70da153441b60 https://github.com/distribusion/supa/pull/21/commits/270b4a01a440646b18751d23e6084324fd646643

@dasnotme Please check the solution.